core/web/presenters/log.go

```diff
@@ -0,0 +1,12 @@
+package presenters
+
+type LogResource struct {
+	JAID
+	LogLevel string `json:"logLevel"`
+	LogSql   bool   `json:"logSql"`
```

Comment:
Let's use a more descriptive name for this like `SqlEnabled` to indicate that it is a `bool`